metadata.yaml

```diff
@@ -11,3 +9,40 @@ summary: |
   Grafana Agent is a telemetry collector for sending metrics, logs, and trace data to the opinionated Grafana observability stack.
 maintainers:
     - Jose Massón <jose.masson@canonical.com>
+
```

Comment:
what do these changes have to do with using a rock image? Am I missing something?

Reply:
These slipped through on the git commit without notice until now, it seems like. Since the [`tox` "render" simply overwrites this file in its entirety](https://github.com/canonical/grafana-agent-k8s-operator/blob/main/tox.ini#L37) it's probably harmless, but I'll put it back out on the next patch